qiskit/circuit/singleton.py

```diff
@@ -64,11 +64,13 @@ def __init__(self, label=None):
    :class-doc-from: class
 .. autoclass:: SingletonGate
    :class-doc-from: class
+.. autoclass:: SingletonControlledGate
+   :class-doc-from: class
```

Comment:
Oops, good catch. I was so used to an `autosummary` approach for the docs I neglected to add this in #10898.